src/Microsoft.Sbom.Api/Exceptions/ClearlyDefinedResponseNotSuccessfulException.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Sbom.Api.Exceptions;
+
+/// <summary>
+/// Exception during argument validation used to indicate when we don't have access to a path passed as argument.
```

Comment:
Seems like the descriptions of these exceptions are all the same. I'd rather have no doc-comments than ones that are misleading or vague.

Reply:
Looks like I accidentally left the old description when I copied the class from one of our other exceptions. I'll change these to the appropriate descriptions.